### PR TITLE
feat: remove use of firstplay event

### PR DESCRIPTION
**Pull request by hugorogz**
## Description
Removed the use of existing 'firstplay' events as it's deprecated in: https://github.com/videojs/video.js/pull/4353

## Specific Changes proposed
Please list the specific changes involved in this pull request.

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

**Review by mister-ben — CHANGES_REQUESTED** [review]
We wouldn't need to trigger `play` instead of `firstplay` -- `play` is already fired so we'll end up with duplicate events.

File: src/js/player.js [hunk 1]
```diff
@@ -1883,7 +1880,6 @@ class Player extends Component {
      * @deprecated As of 6.0 firstplay event is deprecated.
```
Comment (mister-ben) [inline src/js/player.js:1880]: This and other comments relating to `firstplay` can be removed now too.

File: docs/legacy-docs/api/js/doc-data-full.js [hunk 2]
```diff
@@ -6648,24 +6648,6 @@ var docData =
         "longname": "event:seeked",
         "scope": "global"
     },
-    {
```
Comment (gkatsev) [inline docs/legacy-docs/api/js/doc-data-full.js:6651]: this piece should stay as it relates to legacy docs.

**Review by gkatsev — CHANGES_REQUESTED** [review]

**Review by misteroneill — APPROVED** [review]
Looking good to me!

**Review by mister-ben — APPROVED** [review]

**Comment by amtins** [comment]
Just for my understanding, I see that this PR has the **major** label. Does this mean that the `firstplay` event will be removed in version 8?

**Comment by mister-ben** [comment]
That's right

**Pull request merged**